ccsu/cs505/statemanagement/ConcreteStore.java

```diff
@@ -15,27 +14,31 @@
  * this store.
  */
 class ConcreteStore implements Store {
```

Comment:
states instance variable should be private or protected also its type should be the java.util.Map interface from the Collections framework rather than the concrete HashMap class.